InventorySalesApi/Api/Data/IsProductSalableResultInterface.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySalesApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Represents "is product salable" result interface.
+ *
+ * @api
+ */
+interface IsProductSalableResultInterface extends ExtensibleDataInterface
```

Comment:
Please, reuse `\Magento\InventorySalesApi\Api\Data\ProductSalableResultInterface`

Reply:
There is no getSku() method in \Magento\InventorySalesApi\Api\Data\ProductSalableResultInterface, therefore the only way to  understand to which product result applied is to set it in extension attributes. Is it a better option than to introduce new interface?